requirements-dev.txt

```diff
@@ -1,13 +1,13 @@
-coverage==4.5.1
-flake8==3.5.0
-ipdb==0.11
-ipython==7.0.1
-pytest==3.9.1
-pytest-cov==2.6.0
-pytest-sugar==0.9.1
-PyMySQL>=0.9,<=0.9.2
-docker==3.5.1
-sphinx==1.8.1
+coverage==5.1
+flake8==3.7.9
+ipdb==0.13.2
+ipython==7.13.0
+pytest==5.4.1
+pytest-cov==2.8.1
+pytest-sugar==0.9.3
+PyMySQL==0.9.3
+docker==4.2.0
+sphinx==3.0.3
 sphinxcontrib-asyncio==0.2.0
-sqlalchemy==1.2.12
-uvloop==0.11.2; python_version >= '3.5'
+sqlalchemy==1.3.16
+uvloop==0.14.0; python_version >= '3.5'
```

Comment:
If possible should the release pinning be removed or softened. According to those changes it works with later releases anyways. 

This is a major issue for packages which are shipped by distributions because (at least Fedora) usually ships the latest release or not that exact release which is mentioned in the requirements file.